spec/controllers/openid_connect/token_controller_spec.rb

```diff
@@ -79,13 +79,4 @@
       end
     end
   end
-
-  describe '#options' do
-    it 'is empty so it can be used as a pre-flight request' do
-      process :options, 'OPTIONS'
```

Comment:
We have that spec already here: https://github.com/18F/identity-idp/blob/master/spec/requests/openid_connect_cors_spec.rb#L39-L54
I added an expectation for the empty body to that test.